NGitLab/Models/EventAction.cs

```diff
@@ -5,20 +5,32 @@ namespace NGitLab.Models;
 public enum EventAction
 {
     Accepted,
```

Comment:
Is letting few values without EnumMember intended?

Reply:
It is.

I kept the actions not listed [here](https://docs.gitlab.com/ee/user/profile/contributions_calendar.html#user-contribution-events) unchanged, as I wondered whether they truly are recognized by GitLab. If they are not, we should eventually remove them.

Reply:
Not sure that's the only source of truth, but: https://gitlab.com/gitlab-org/gitlab/-/tree/master/config/events